### PR TITLE
add jvm clusterdashbaord

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

- add jvm clusterdashbaord

### CODE REVIEW

**Comment by benjaminhuo** [comment]
Please add namespaced dashboard as well

**Comment by zhu733756** [comment]
> Please add namespaced dashboard as well

fixed

File: contrib/gallery/jvm.yaml [hunk 1]
```diff
@@ -0,0 +1,231 @@
+apiVersion: monitoring.kubesphere.io/v1alpha2
+kind: ClusterDashboard
```
Comment (benjaminhuo) [inline contrib/gallery/jvm.yaml]: namespaced crd is ClusterDashboard ?

Comment (zhu733756) [reply contrib/gallery/jvm.yaml]: fixed, have not saved in local enviroment.

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**